лабораторная4/task1.py

```diff
@@ -0,0 +1,107 @@
+class Earphones:
+    """
+            Создание и подготовка к работе объекта "Наушники"
+            :param name: Марка, защищенный аргумент, пользователь не может создать новые виды наушников
+            :param manufacturer: Производитель, защищенный аргумент, пользователь не может создать новые виды наушников
+    """
+    def __init__(self, name: str, manufacturer: str):
+        self._name = name
+        self._manufacturer = manufacturer
+
+    @property
+    def name(self):
+        return self._name
+
+    @property
+    def manufacturer(self):
+        return self._manufacturer
+
+    @name.setter
+    def name(self, new_name: str):
+        self._name = new_name
+
+    @manufacturer.setter
+    def manufacturer(self, new_manufacturer: str):
+        self._manufacturer = new_manufacturer
+
+    def __str__(self):
+        return f'Наушники марки "{self._name}" производителя "{self._manufacturer}"'
+
+    def __repr__(self):
+        return f'{self.__class__.__name__}: name={self._name!r}, manufacturer ={self._manufacturer})'
+
+class Wired_Earphones(Earphones):
+    """
+                Создание и подготовка к работе объекта "Проводные наушники"
+                :param name: Марка, защищенный аргумент, пользователь не может создать новые виды наушников
+                :param manufacturer: Производитель, защищенный аргумент, пользователь не может создать новые виды наушников
+                :param condition: Состояние, ради избежания ошибок вводится через setter
+    """
+    def __init__(self, name: str, manufacturer: str, condition: str):
+        super().__init__(name, manufacturer)
+        if condition != 'plugged' and condition != 'unplugged':
+            raise ValueError('Может принимать только значение plugged или unplugged')
+        self.condition = condition
+
+    def plug(self):
+        condition = 'plugged'

```

Comment:
```suggestion
        self.condition = 'plugged'

```
Вы же обновляете переменную текущего экземпляра, а без `self` вы просто создаете новую переменную, причем локальную, которая никак не связан с экземпляром класса